modules/nf-core/bismark/methylationextractor/main.nf

```diff
@@ -24,16 +24,25 @@ process BISMARK_METHYLATIONEXTRACTOR {
 
     script:
     def args = task.ext.args ?: ''
```

Comment:
Should there be a way to prefix the output files here?

Reply:
I don't really understand what you mean.. Bismark already sets the output files with a prefix based on the input filenames..

Reply:
Sorry, read your comments in the wrong answer. I'll check..

Reply:
Can't see an option for this in Bismark (see [source](https://github.com/FelixKrueger/Bismark/blob/430df15af5b52686487d4eeb6a97d1b2390bbf96/bismark_methylation_extractor#L959-L994)).

There is `--output_dir`, should I use the prefix for that if set?

Reply:
That would mess with the output capture since it doesn't recurse down. 

I guess the question on whether this should have a prefix depends on if the workflow will process groups of samples and if the files are passed along for further batch processing, will this then create a filename collision? If this is a possibility, then the files might need to be renamed based on the prefix. If the tool should not be used like this, then just leave it out.

Reply:
I don't think it'll be a problem - extraction always happens after deduplication so on complete samples. As such, I've not had to worry about filename collisions in the past.